builder/builder.go

```diff
@@ -569,27 +572,64 @@ func (b *Builder) getUpstreamBundles(ver string, prune bool) error {
 }
 
 // getBundlePath returns the path to the bundle definition file for a given
-// bundle name, or error if it cannot be found. Looks first in local-bundles,
-// then upstream-bundles.
+// bundle name, or error if it cannot be found. Looks in the following order:
+// local-bundles/
+// local-packages
+// upstream-bundles/clr-bundles-<ver>/bundles/
+// upstream-bundles/clr-bundles-<ver>/packages
 func (b *Builder) getBundlePath(bundle string) (string, error) {
 	// Check local-bundles
 	path := filepath.Join(b.LocalBundleDir, bundle)
 	if _, err := os.Stat(path); err == nil {
 		return path, nil
 	}
 
+	pwd, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+	// Check local-packages
+	path = filepath.Join(pwd, "local-packages")
+	if len(localPackages) == 0 {
+		if _, err := os.Stat(path); err == nil {
+			localPackages, err = parsePackageBundleFile(path)
+			if err != nil {
+				return "", err
+			}
+		}
+	}
```

Comment:
You have three places (here, L614, and L788) with these same 8 lines. I'd pull that out into a standalone `getPackageList(filename string) (map[string]bool, err)` function.

Reply:
good call, will do.

Reply:
Done